themes/hugo-rladies/layouts/events/single.html

```diff
@@ -42,7 +33,19 @@ <h4 class="my-2">{{ .n_events_past }}</h4>
 {{ end }}
 
 {{ define "footer" }}
-<script type="text/javascript">
+  {{ $js := slice
```

Comment:
this is nice! for the rOpenSci website we minified externally but this is cleaner.